tonic/models/vic/grid_params.py

```diff
@@ -602,6 +732,71 @@ def __init__(self, organic_fract=False, spatial_frost=False,
                                'basin_depth': 'm',
                                'basin_area': 'fraction'}
 
+        # State Variables
+        self.state = {'gridcell': 'N/A',
+                      'Soil_moisture': 'mm',
+                      'Soil_ice': 'mm',
+                      'Canopy_water': 'mm',
+                      'Snow_age': 'time steps',
```

Comment:
Yes, I agree.  No, the sim timestep is not written to the state files as of VIC 4.2.  Do you want me to have tonic change its units?  If so, the state file won't work with VIC...

If you want to make that change to tonic as part of this commit, I could add the timestep as an input argument to tonic (only required if converting a state file) and convert Snow_age from time steps to sec.  But I'm guessing you'd want to wait and include that in a commit that is linked to a similar change on the VIC side.